### PR TITLE
Updated the requirements for all environments

### PR DESCRIPTION
**Pull request by harshad16**
Updated the requirements for all environments
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -70,6 +70,17 @@ If you wish to build container images locally, you can do so (an example for
...
+Updating the dependencies
+=========================
+
+This script can be used for updating the requirements for each of the environments.
+
+Execute command: `./update-requirements.sh <sub-directories>`
+
+Example: `./update-requirements.sh ubi8-py39 f34-py39`
+
+Prerequisite: Pipenv, micropipenv and different python ecosystem
```
Comment (fridex) [inline README.rst:82]:
```suggestion
Prerequisite: Pipenv, micropipenv and different Python interpreter versions
```

**Review by fridex — COMMENTED** [review]
one minor wording, otherwise LGTM 👍🏻

File: README.rst (hunk 1 above)
Comment (harshad16) [reply README.rst]: thanks frido :)

**Comment by harshad16** [comment]
Merging and releasing a new version
thanks for the reviews

**Pull request merged**